tests/test_set_ocean_surface_temperature.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+import os
+import numpy as np
+import fv3gfs.wrapper
+import fv3gfs.util
+from mpi4py import MPI
+from util import (
+    get_default_config,
+    get_state_single_variable,
+    main,
+    replace_state_with_random_values,
+)
+
+
+test_dir = os.path.dirname(os.path.abspath(__file__))
+
+
+def mask_non_ocean_values(field):
+    is_ocean = np.isclose(get_state_single_variable("land_sea_mask"), 0.0)
+    return np.where(is_ocean, field, np.nan)
```

Comment:
Optional minor improvements:

`return field[is_ocean]` would be more efficient for the way you're using this. `np.where` will give you a full array with nans for masked values, whereas `field[is_ocean]` will give you a 1D array containing only the non-masked values. Since maintaining the array shape isn't important here, you could opt for the cheaper option.

You could also avoid retrieving the land-sea mask multiple times by taking in `mask_non_ocean_values(*fields):` and then iterating over the fields after retrieving the land_sea mask.

These are not really necessary changes, so only if you're interested.

Reply:
Thanks -- I implemented both of these suggestions.